core/scripts/Voting.js

```diff
@@ -71,28 +84,43 @@ class VotingSystem {
 
     // Persist the vote only after the commit hash has been sent to the Voting contract.
     await this.writeVote(request, roundId, { price: fetchedPrice.toString(), salt: salt.toString() });
+    return true;
   }
 
   async runReveal(request, roundId) {
     const persistedVote = await this.readVote(request, roundId);
     // If no vote was persisted and committed, then we can't reveal.
-    if (persistedVote) {
-      await this.voting.revealVote(request.identifier, request.time, persistedVote.price, persistedVote.salt, {
-        from: this.account
-      });
+    if (!persistedVote) {
+      return false;
     }
+    await this.voting.revealVote(request.identifier, request.time, persistedVote.price, persistedVote.salt, {
+      from: this.account
+    });
+    return true;
   }
 
   async runIteration() {
     const phase = await this.voting.getVotePhase();
     const roundId = await this.voting.getCurrentRoundId();
     const pendingRequests = await this.voting.getPendingRequests();
+    const updatesMade = [];
     for (const request of pendingRequests) {
+      let didUpdate = false;
       if (phase == VotePhasesEnum.COMMIT) {
-        await this.runCommit(request, roundId);
+        didUpdate = await this.runCommit(request, roundId);
       } else {
-        await this.runReveal(request, roundId);
+        didUpdate = await this.runReveal(request, roundId);
       }
+      if (didUpdate) {
+        updatesMade.push(request);
+      }
+    }
+    if (updatesMade.length > 0) {
+      // TODO(ptare): Get email copies.
```

Comment:
What does this mean?

Reply:
copy = actual final text. When we have the email copies, we'll need to plug them in here.

Reply:
Still confused - maybe we can discuss IRL.

Reply:
Discussed offline, going to fill these in in the next few days.